bundles/org.openhab.binding.tado/src/main/java/org/openhab/binding/tado/internal/handler/TadoBatteryChecker.java

```diff
@@ -57,37 +58,23 @@ private synchronized void refreshZoneList() {
             Long homeId = homeHandler.getHomeId();
             if (homeId != null) {
                 logger.debug("Fetching (battery state) zone list for HomeId {}", homeId);
+                zoneList.clear();
                 try {
-                    zoneList = homeHandler.getApi().listZones(homeId);
+                    homeHandler.getApi().listZones(homeId).forEach(zone -> {
+                        boolean batteryLow = !zone.getDevices().stream().map(ControlDevice::getBatteryState)
+                                .filter(Objects::nonNull).allMatch(s -> s.equals("NORMAL"));
+                        zoneList.put(Long.valueOf(zone.getId()), OnOffType.from(batteryLow));
+                    });
                 } catch (IOException | ApiException e) {
-                    zoneList = null;
-                    logger.debug("Fetch (battery state) zone list exception", e);
+                    logger.debug("Fetch (battery state) zone list exception");
```

Comment:
Did you remove the stacktrace by intention?

Reply:
Yes. :)